lib/server.js

```diff
@@ -112,6 +121,15 @@ module.exports = function createServer(config, log) {
       }
       email = normalizedEmail(email)
 
+      // Check blocklist before retrieving records, save a lookup
```

Comment:
> this will bypass the logic that checks for allowedIPs and allowedEmails

Good point! I had not thought about that. Will revise the logic for this.